src/compliance/compliance-requirements.controller.ts

```diff
@@ -266,4 +266,29 @@ export class ComplianceRequirementsController {
     const result = await this.complianceRequirementsService.modify(ticker, id, params);
     return handleServiceResult(result);
   }
+
+  @ApiOperation({
+    summary: 'Check if the requirements are paused',
+    description:
+      'This endpoint checks if the compliance requirements are paused for a given ticker',
+  })
+  @ApiParam({
+    name: 'ticker',
+    description: 'The ticker of the Asset whose compliance requirements status are to be fetched',
+    type: 'string',
+    example: 'TICKER',
+  })
+  @ApiOkResponse({
+    description: 'Compliance Requirement status',
+    schema: {
+      type: 'boolean',
+    },
+  })
+  @ApiNotFoundResponse({
+    description: 'The Asset does not exist',
+  })
+  @Get('are-paused')
```

Comment:
I'd say something like:
```suggestion
  @Get('/status')
```
and make the return an object with a key.

Generally we want to routes to be REST like, with the exception being for actions.